### PR TITLE
Move 'verbose-migration' to core

### PR DESCRIPTION
**Pull request by jimbru**
Moves `verbose-migration` into `core` as a convenience for building programmatic migrations.

Related to issue #13.

### CODE REVIEW

File: ragtime.core/src/ragtime/main.clj [hunk 1]
```diff
@@ -35,7 +25,7 @@
   (let [db (core/connection database)]
     (doseq [m (resolve-migrations migrations)]
       (core/remember-migration m))
-    (core/rollback-last db (or (when n (Integer/parseInt n)) 
+    (core/rollback-last db (or (when n (Integer/parseInt n))
```
Comment (weavejester) [inline ragtime.core/src/ragtime/main.clj]: What does this have to do with verbose migrations?

Comment (jimbru) [inline ragtime.core/src/ragtime/main.clj]: Looks like it trimmed extra whitespace at EOL.

Comment (weavejester) [inline ragtime.core/src/ragtime/main.clj]: Ah right, I saw the change and thought it had added the "1" at the end. False alarm :)

**Comment by drusellers** [comment]
:+1:

**Comment by weavejester** [comment]
This functionality has been exposed from 0.4.0 onward in the `ragtime.repl` namespace.
